shell/platform/windows/flutter_windows_view.h

```diff
@@ -46,8 +46,23 @@ class FlutterWindowsView : public WindowBindingHandlerDelegate {
   // Get the view's unique identifier.
   FlutterViewId view_id() const;
 
-  // Creates rendering surface for Flutter engine to draw into.
-  // Should be called before calling FlutterEngineRun using this view.
+  // Whether this view is the implicit view.
+  //
+  // The implicit view is a special view for backwards compatibility.
+  // The engine assumes it can always render to this view, even if the app has
+  // destroyed the window for this view. The embedder must ignore presents to
+  // this view after it is destroyed.
+  //
+  // The implicit view is the only view that can be created before the
+  // engine is launched.
+  //
+  // See:
+  // https://api.flutter.dev/flutter/dart-ui/PlatformDispatcher/implicitView.html
+  bool IsImplicitView() const;
+
+  // Create a rendering surface for Flutter engine to draw into.
+  //
+  // This is a no-op if using software rasterization.
   void CreateRenderSurface();
```

Comment:
Why is this method public?

Reply:
This is public to the Windows embedder but not end users. From what I can tell, this design (which predates me and multi-view) was chosen to make unit testing easier.

A potentially better design would be to have a protected constructor for unit testing and a `FlutterWindowsView::Create` static method which calls the constructor, calls `CreateRenderSurface`, and returns the view. Such a refactoring would be out-of-scope for this change though :)